source/browseMode.py

```diff
@@ -204,12 +204,36 @@ class BrowseModeTreeInterceptor(treeInterceptorHandler.TreeInterceptor):
 	def _get_currentNVDAObject(self):
 		raise NotImplementedError
 
+	ALWAYS_SWITCH_TO_PASS_THROUGH_ROLES = (
+		controlTypes.ROLE_COMBOBOX,
+		controlTypes.ROLE_EDITABLETEXT,
+		controlTypes.ROLE_LIST,
+		controlTypes.ROLE_SLIDER,
+		controlTypes.ROLE_TABCONTROL,
+		controlTypes.ROLE_MENUBAR,
+		controlTypes.ROLE_POPUPMENU,
+		controlTypes.ROLE_TREEVIEW,
+		controlTypes.ROLE_TREEVIEWITEM,
+		controlTypes.ROLE_SPINBUTTON,
+		controlTypes.ROLE_TABLEROW,
+		controlTypes.ROLE_TABLECELL,
+		controlTypes.ROLE_TABLEROWHEADER,
+		controlTypes.ROLE_TABLECOLUMNHEADER)

```

Comment:
Convention for long collections is to treat the last item like every other item and put the closing parenthesis on its own line. So:

```
		controlTypes.ROLE_TABLECOLUMNHEADER,
	)
```

(The trailing comma is not a mistake.)

This means that adding a new entry at the bottom just adds a new line, rather than changing an unrelated line as well. For more on this, see [this Python FAQ](https://docs.python.org/2/faq/design.html#why-does-python-allow-commas-at-the-end-of-lists-and-tuples) and [this StackOverflow thread](http://stackoverflow.com/questions/29063868/does-the-extra-comma-at-the-end-of-a-dict-list-or-set-has-any-special-meaning).